docker/bin/msfvenom

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# determine script path
+pushd $(dirname $(dirname $(dirname $(realpath $0)))) > /dev/null
```

Comment:
alright. lets try `readlink` looks like that resolves correctly on both systems.